File: web3rpc/rpc-specs/code-samples/javascript/klay/filter/getFilterLogs.js
const { JsonRpcProvider } = require("@kaiachain/ethers-ext");
(() => {
  const provider = new JsonRpcProvider("https://public-en-kairos.node.kaia.io");
  const quantity = "0x16";
  provider.klay
    .getFilterLogs(quantity, {}, (err, data, response) => {})
    .then((data) => {
      console.log(data);
    });
})();

File: web3rpc/rpc-specs/code-samples/python/klay/filter/getFilterLogs.py
from web3 import Web3
from web3py_ext import extend

host = "https://public-en-kairos.node.kaia.io"

quantity = "0x16"

w3 = Web3(Web3.HTTPProvider(host))
klay_response = w3.klay.get_filter_logs(quantity)

print(klay_response)
